src/components/graph/graph.ts

```diff
@@ -113,7 +113,7 @@ export default class GraphComponent {
   private _startNode: null | GraphNodeId = null
   private _targetNode: null | GraphNodeId = null
 
-  private _homeLocation: IHomeLocation = {x: 0, y: 0, scale: 1}
+  private _homeLocation: ITransform = {translation: {x: 0, y: 0}, scale: 1}
```

Comment:
The issue here is `{translation: {x, y}, scale}` is not of type ITransform`. It's an extension of `ITransform`. so I think the current state is a bit confusing if you look without prior understanding of what's going on.